### ISSUE TITLE
[Accessibility] WCAG 2.1 aa Criterion 3.2.2 (change of focus) issue for card form. v.6.19

### ISSUE THREAD

**Issue by captador** [Adyen/adyen-web#3436]
**Describe the bug**
Card form moves focus from expiry date field to security code automatically when card expiration is filled. It goes against Criterion 3.2.2  of AA scope 

![Image](https://github.com/user-attachments/assets/c38987a9-c5f1-4204-af96-0286583460fc)

**To Reproduce**
1. Have web form for card payment
2. Fill up expiry date with 03/30 value
3. Spot that focus moves to security code

**Expected behavior**
No automatic focus jump

**Screenshots**

**Desktop (please complete the following information):**
 - MacOS
 - Chrome
 - 138.0.7204.184

**Additional context**
- "@adyen/adyen-web": "6.19.0"

**Comment by m1aw** [comment]
Hey @captador 

Thank you for the feedback, this has been mentioned in the past from other merchants, and we are discussing how to deal with it internally. We don't consider it violates 3.2.2 (we have consulted with our a11y partner in the past), but since has been reported a few times we are thinking on changing. 

This is however a bit of a balance between UX and a11y, so we want to be careful with these changes.

For now you can pass the `autoFocus: false` property/value to the card component as [explained in our documentation](https://docs.adyen.com/payment-methods/cards/web-component/#optional-configuration).

If this isn't enough feel free to re-open the issue happy to discuss further.
